environment.md

```diff
@@ -1,19 +1,35 @@
-# Environment
+# Server Environment
 
-WordPress will run in a minimum server environment. However, WordPress doesn't run optimally on the minimum system requirements. This section will cover the _recommended_ server environment for WordPress to run more effectively.
+Although WordPress can work in almost any environment, even very minimal ones, it must be acknowledged that it does not work completely well in these. That's why here we are going to make some minimum recommendations of the environment in which it would work most effectively.
 
-## Recommended Servers
+## Web Server
 
-WordPress recommends using one of these servers:
+The web server is the system where the files of the web site are hosted and where the users arrive to consult them. There are many web servers, and in principle anyone who supports a PHP connection should be able to work with WordPress.
```

Comment:
"The web server is the system where the files of the web site are hosted and where the users arrive to consult them." 
Trying to think of a better way to put this. The web server doesn't store any files, but it is what serves files to users.

Reply:
Updated the text:

The web server is the software dedicated to run the files of the website and where users come to consult them, mainly through the Web.

There are many web servers and, in principle, anyone who supports the execution of PHP files should be able to work with WordPress.

Reply:
Yeah, it's so hard to describe a web server. I think it's fine to start with this, and if we think of a better description, can iterate.

For the second sentence, maybe this -- since "anyone" usually means a person?
"There are many web servers and, generally, any that support the execution of PHP files should be able to work with WordPress."

Reply:
fixed